book_source/03_intermediate_users_guide/01_pecan_xml.Rmd

```diff
@@ -1,195 +1,487 @@
+	</config.header>
+</model>
 ```
 
-* **url** : [required] endpoint for Brown Dog to be used.
-* **username** : [optional] username to be used with the endpoint for Brown Dog.
-* **password** : [optional] password to be used with the endpoint for Brown Dog.
+Some important tags are as follows:
 
+- `id` -- The unique numeric ID of the model in the Bety `models` table. If this is present, then `type` and `binary` are optional since they can be determined from Bety.
+- `type` -- The model "type", matching the Bety `modeltypes` table `name` column. This also refers to which PEcAn model-specific package will be used. In PEcAn, a "model" refers to a specific version (e.g. release, git commit) of a specific model, and "model type" is used to link different releases of the same model. Model "types" also have specific PFT definitions and other requirements associated with them (e.g. the ED2 model "type" requires a global land cover database).
+- `binary` -- The file path to the model executable. If omitted, PEcAn will use whatever path is registered in Bety for the current machine. 
+- `job.sh` -- Additional options added to the `job.sh` script, which is used to execute the model. This is useful for setting specific environment variables, load modules, etc.
 
-## PFT Selection
+This information is currently used by the following PEcAn workflow function:
 
-The PEcAn system requires at least 1 PFT (Plant Functional Type) to be specified inside the `<pfts>` section. 
+`write.configs()`, `run.models()`
 
-#### functions:  
+#### Model-specific configuration {#xml-model-specific}
 
-`get.traits()`
+See the following:
 
+* [ED2][ED2 Configuration]
+* [SIPNET][SIPNET Configuration]
+* [BIOCRO][BioCro Configuration]
 
-#### tags
+#### ED2 specific tags {#xml-ed}
 
+Following variables are ED specific and are used in the [ED2 Configuration](ED2-Configuration).
+
+Starting at 1.3.7 the tags for inputs have moved to `<run><inputs>`. This includes, veg, soil, psscss, inputs.
 
 ```xml
-<pfts>
-	<pft>
-		<name>sipnet.temperate.coniferous</name>
-		<outdir>/home/carya/testrun.pecan/pft/1/</outdir>
-		<constants>
-			<num>1</num>
-		</constants>
-	</pft>
-</pfts>
+	<edin>/home/carya/runs/PEcAn_4/ED2IN.template</edin>
+	<config.header>
+		<radiation>
+			<lai_min>0.01</lai_min>
+		</radiation>
+		<ed_misc>
+			<output_month>12</output_month>
+		</ed_misc> 
+	</config.header>
+	<phenol.scheme>0</phenol.scheme>
 ```
 
-* **name** : [required] the pft as is found inside the BETY database, this needs to be an exact match.  
-* **outdir**: [optional] path in which pft-specific output will be placed during meta-analysis and sensitivity analysis. If not specified it will be written into `<outdir>/<pftname>`.  
-* **contants**: [optional] this section contains information that will be written directly into the model specific configuration files. PEcAn does not look at the information in this section.  
+  
+* **edin** : [required] template used to write ED2IN file
```

Comment:
The edin tag is optional, not required. PEcAn will grab the default from PEcAn.ED2 R package if none is specified.